terraform-scripts/.terraform.lock.hcl

```diff
@@ -2,7 +2,7 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/integrations/github" {
-  version     = "5.33.0"
+  version     = "5.34.0"
```

Comment:
You can't edit this manually, this is generated by the `terraform init --upgrade` command